### PR TITLE
Metldata & MASS API update

### PR DESCRIPTION
**Pull request by ac-jorellanaf**
- Updated to metadata model 1.0.0 schema.
- Removed plenty of the edge case objects to be re-added later.
- More changes probably necessary, but waiting on metldata updates.
- Removed metadata search and repo services from docker compose

### CODE REVIEW

File: src/components/browse/dataset/datasetAccordion/datasetSummary/datasetSummary.tsx [hunk 1]
```diff
@@ -67,7 +63,7 @@ const DatasetSummary = (props: dataSetDetailsProps) => {
                   </Col>
                 </Row>
               </Button>
-              {props.hit.content.ega_accession !== null ? (
+              {/* {props.hit.content.ega_accession !== null ? (
```
Comment (Cito) [inline src/components/browse/dataset/datasetAccordion/datasetSummary/datasetSummary.tsx:66]: I think we can get the EGA accession (if one exists) from the xref property.

File: src/components/browse/singleDatasetView/singleDatasetView.tsx [hunk 2]
```diff
@@ -130,7 +131,7 @@ const SingleDatasetView = () => {
                 handleOpen={handleOpen}
                 classes="float-end ms-2 ms-sm-4 "
               />
-              {details.ega_accession !== null ? (
+              {/* {details.ega_accession !== null ? (
```
Comment (Cito) [inline src/components/browse/singleDatasetView/singleDatasetView.tsx:134]: See above.

**Review by Cito — CHANGES_REQUESTED** [review]
This is a good starting point.

Can you review the the commented out parts? If they are not needed any more, remove them. If something needs to be done or clarified, add a small TBD or TODO comment explaining what is still missing or unclear.

File: src/components/browse/dataset/datasetAccordion/datasetSummary/datasetSamples.tsx [hunk 3]
```diff
@@ -48,19 +49,17 @@ const DatasetSamples = (props: dataSetSamplesProps) => {
                   {props.samples.stats?.sex?.unkown}
```
[the remark this answers is not in the document]
Comment (ac-jorellanaf) [reply src/components/browse/dataset/datasetAccordion/datasetSummary/datasetSamples.tsx:49]: That was a typo in the metadata repo I'm afraid.

File: src/mocks/responses.js [hunk 4]
```diff
@@ -41,13 +41,13 @@ const localResponses = {
...
-  "GET http://127.0.0.1:8002/datasets/*?embedded=true": embeddedDataset,
+  "GET http://127.0.0.1:8002/artifacts/embedded_public/classes/EmbeddedDataset/resources/*": embeddedDataset,
 
   // Get summary data from a single dataset
-  "GET http://127.0.0.1:8002/dataset_summary/*": datasetSummary,
+  "GET http://127.0.0.1:8002/artifacts/stats_public/classes/DatasetStats/resources/*": datasetSummary,
 
   // Get summary data from entire metadata database
-  "GET http://127.0.0.1:8092/metadata_summary/": metadataSummary,
+  "GET http://127.0.0.1:8002/artifacts/stats_public/classes/GlobalStats/resources/*": metadataSummary,
```
Comment (Cito) [inline src/mocks/responses.js:50]: it's just `/stats` (the global stats are not delivered as an artifact)

**Review by Cito — CHANGES_REQUESTED** [review]
Looks good, just one url to change.

**Review by Cito — APPROVED** [review]

**Pull request merged**